src/microbe_stage/MicrobeHUD.cs

```diff
@@ -611,7 +623,7 @@ private void UpdateHoverInfo()
                     compoundsLabel.Valign = Label.VAlign.Center;
                     hoveredItems.AddChild(compoundsLabel);
                     compoundsLabel.AddConstantOverride("line_spacing", -5);
-                    compoundsLabel.Text = "Compounds: \n";
+                    compoundsLabel.Text = TranslationServer.Translate("MICROBE_HUD_COUMPOUNDS");
```

Comment:
I mentioned this before, but I'm a bit uncertain if going this far to separate out things is necessary
Requiring translators to fill in a bunch of different things that all translate to `Compounds:` wouldn't be super good
something like COMPOUNDS_LIST would be more general for use in all places where this kind of translation is needed.